sql/src/main/java/io/druid/sql/calcite/planner/Calcites.java

```diff
@@ -107,6 +108,7 @@ public static SchemaPlus createRootSchema(final Schema druidSchema, final Author
 
   public static String escapeStringLiteral(final String s)
   {
+    Preconditions.checkNotNull(s);
     if (s == null) {
```

Comment:
This check immediately after Preconditions.checkNotNull?

Reply:
removed the == null check, it was not needed.